ONSdigital/ras-secure-message
# Ts 235 inform case service when new secure message sent

**What is the context of this PR?**

- Link to Trello card.
[Card1](https://trello.com/c/Evs6fcKH) [Card2](https://trello.com/c/x7k5C9SY)

**How to review**

- Make sure the system uses the mock and the real service (can be seen with party service also).
- The tests in test_case_service_integration and test_party_service_integration are skipped, unskip them and make sure they run correctly.
- Make sure all tests run and there are no issues with secure messaging.
